src/orion/core/worker/experiment.py

```diff
@@ -633,8 +633,17 @@ def _branch_config(self, conflicts, branching_configuration):
         """
         experiment_brancher = ExperimentBranchBuilder(conflicts, branching_configuration)
 
-        if not experiment_brancher.is_resolved or experiment_brancher.auto_resolution:
+        needs_manual_resolution = (not experiment_brancher.is_resolved or
+                                   experiment_brancher.auto_resolution)
```

Comment:
If auto_resolution is True, wouldn't that mean that the user does not want the manual resolution?

Reply:
This is confusing indeed. If using auto-resolution, then the conflict resolver prompts so that the user confirms that the changes are fine.

Reply:
I see